### PR TITLE
homepage update

### PR DESCRIPTION
**Pull request by Houshilian**

### CODE REVIEW

File: src/components/HomePage/HomePage.css [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+input { 
+  margin: auto 5px;
+}
+.submitbtn{
+  height: 30px;
+  padding: 4px 16px;
+  color:rgba(0, 0, 0, 0.87);
+  background: rgba(0, 0, 0, 0.04);
+  border-radius: 4px;
+  position: absolute;
```
Comment (tiandin) [inline src/components/HomePage/HomePage.css:18]: Will the absolute position make our UIs lose their flexibility? More specifically, will the component adapt into different size of the windows?

File: src/components/HomePage/HomePage.js [hunk 2]
```diff
@@ -112,11 +116,16 @@ class HomePage extends Component{
...
 
-
     //Single
-    const url = this.generateURL(cursor, projectID);
-    const res = await axios.get(url);
-    this.processData(res.data.hits);
+    try {
+      const url = this.generateURL(cursor, projectID);
+      const res = await axios.get(url);
+      console.log("test", res.status);
```
Comment (tiandin) [inline src/components/HomePage/HomePage.js:123]: Please remove the console log, those are just use for testing, we should not keep them when raise a pull request

File: src/components/HomePage/HomePage.js [hunk 3]
```diff
@@ -112,11 +116,16 @@ class HomePage extends Component{
...
+      this.processData(res.data.hits);
+    } catch (error) {
+      console.log(error);
```
Comment (tiandin) [inline src/components/HomePage/HomePage.js:126]: Same as above

File: src/components/SamplePage/SamplePage.js [hunk 4]
```diff
@@ -4,27 +4,53 @@ import { IconButton, Stack } from "@mui/material";
...
+        super(props);
+        this.state={
+            validatePass: true,
+            errorMessage: '',
+        };
+    }
+
+    //handle the submit action 
+    submitForm =(event) => {
+        // const { item, onFormSubmit } = this.props;
```
Comment (tiandin) [inline src/components/SamplePage/SamplePage.js:18]: Please remove the commented codes, if you don't need them

**Review by tiandin — CHANGES_REQUESTED** [review]
Please resolve my comments

File: src/components/HomePage/HomePage.js (hunk 2 above)
Comment (Houshilian) [reply src/components/HomePage/HomePage.js]: Remove the console log, see 4db17c7

File: src/components/HomePage/HomePage.js (hunk 3 above)
Comment (Houshilian) [reply src/components/HomePage/HomePage.js]: Same as above

File: src/components/HomePage/HomePage.css (hunk 1 above)
Comment (Houshilian) [reply src/components/HomePage/HomePage.css]: change absolute position, see 10446c8

File: src/components/SamplePage/SamplePage.js (hunk 4 above)
Comment (Houshilian) [reply src/components/SamplePage/SamplePage.js]: remove submitForm, see 4da0ae8

**Review by tiandin — APPROVED** [review]
LGTM

**Pull request merged**